apps/modernization-ui/src/pages/patient/profile/administrative/AdministrativeTable.tsx

```diff
@@ -132,9 +130,55 @@ export const AdministrativeTable = ({ patient }: Props) => {
                 break;
         }
     };
+
+    const generateTableRow = (administrative: PatientAdministrative, index: number): TableBody => {
+        return {
+            id: index,
+            tableDetails: [
+                {
+                    id: 1,
+                    title: administrative?.asOf ? internalizeDate(administrative?.asOf) : null
+                },
+                {
+                    id: 2,
+                    title: administrative?.comment
+                },
+                {
+                    id: 3,
+                    title: (
+                        <div className="table-span">
+                            <Button
+                                type="button"
+                                unstyled
+                                disabled={patient?.status !== 'ACTIVE'}
+                                onClick={() => setIsActions(isActions === index ? null : index)}>
+                                <Icon.MoreHoriz className="font-sans-lg" />
+                            </Button>
+
+                            {isActions === index && (
+                                <ActionState
+                                    notDeletable
+                                    handleOutsideClick={() => setIsActions(null)}
```

Comment:
<img width="1095" alt="Screenshot 2023-12-13 at 12 09 56 AM" src="https://github.com/CDCgov/NEDSS-Modernization/assets/125376056/86e85eb2-4632-477c-8c7d-f629180fc05c">

Can still has delete option if view is clicked. But Button does not show